tests/baselines/reference/contextuallyTypedStringLiteralsInJsxAttributes02.errors.txt

```diff
@@ -84,7 +84,7 @@ tests/cases/conformance/types/contextualTypes/jsxAttributes/file.tsx(36,44): err
 !!! error TS2769:     Type '{ extra: true; goTo: string; }' is not assignable to type 'IntrinsicAttributes & ButtonProps'.
 !!! error TS2769:       Property 'extra' does not exist on type 'IntrinsicAttributes & ButtonProps'.
 !!! error TS2769:   Overload 2 of 2, '(linkProps: LinkProps): Element', gave the following error.
-!!! error TS2769:     Type '{ extra: true; goTo: "home"; }' is not assignable to type 'IntrinsicAttributes & LinkProps'.
+!!! error TS2769:     Type '{ extra: true; goTo: string; }' is not assignable to type 'IntrinsicAttributes & LinkProps'.

```

Comment:
At first, I assumed that this is an acceptable change but I rechecked the equivalent call using a call expression and the reported error there doesn't contain a widened string:
[TS playground](https://www.typescriptlang.org/play?ts=5.0.0-dev.20221117#code/JYWwDg9gTgLgBAJQKYEMDG8BmUIjgcilQ3wChSkAPSWOYAOxiSk3STgGEAbYNAaxQAjLkgAKOMAGc4Ab1JwFcNAAtgXACZF6AfgBccSTCgMA5gG55itFxSTJAORQgkeg0dMWAvuSo14DJhY2OAAhAFcYGAh6cQgpOComenVpbl4BYTEJaTlFOGi0-n0ACj59ACIRTBhyuAAfOHLjE2UagEo4AF4APjgANwhgdS8famh-RmZWNHYAGQY+WPjEpGTUnn4hESWcywUTCAAVCArlXCRahvK0aJh0GpGKMdp1JGsUIjhMMPoMYGi4ABZFAMcKRaLFQQRKIxbL6MEwnZtfQAKQAygANAB0AFERM5GBZfOM4K93p9vr8YP96ECQfQERCePRFnC4PMWUjUZjcfjVjALOQbvRDHBBABmLpwAA8wNB0IBMixypkB2OunKZ2c5U8ngSlCMKDgAHpuoLhaKAO7AGDKDgoLhcKVyhkK+jFGRwZVY1VHE4ELVIfB6xJQFD6IxhdieNpAA)

Gonna have to take another look at this to understand why JSX and a regular call different in behavior for this case after my change.